gst/tensor_filter/tensor_filter.c

```diff
@@ -296,6 +297,10 @@ gst_tensor_filter_class_init (GstTensorFilterClass * klass)
       g_param_spec_string ("outputtype", "Output tensor element type",
           "Type of each element of the output tensor ?", "",
           G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
+  g_object_class_install_property (gobject_class, PROP_NNAPI,
+      g_param_spec_boolean ("use-nnapi", "Use NNAPI",
+          "Choose if it uses NNAPI or not", FALSE,
+          G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
```

Comment:
Make this general to other NNFWs.
PROP_FWOPTION (Framework Option) would be better, which can set "GPU=XX" or any other options for Caffe, PyTorch, TF, TF-LITE or anyone else.
And then, implement some parsing rules, apply it to NNAPI.

Reply:
Hm.. sorry but I'm afraid I don't understand what you say. I will find you soon.